### ISSUE TITLE
Stutter on initial playback with Plex

### ISSUE THREAD

**Issue by diblaze** [ErsatzTV/ErsatzTV#1765]
Loading a stream within Plex always plays flawlessly the first five seconds then it’s stutters/hangs for about 5 to 6 seconds and then it starts playing again without issue.
Loading the stream with VLC does not have this behavior.

what kind of debug logs do I need to post, I guess from both Plex and Erzats?

**Comment by raknam** [comment]
I think this can come from Plex and its buffering/recording system, especially if VLC can read the same stream directly. I hate saying this, but could you also ask for their support?

**Comment by daytonturner** [comment]
@diblaze I'm finding the same, did you ever find a workaround for this?

**Comment by diblaze** [comment]
> @diblaze I'm finding the same, did you ever find a workaround for this?

No, I never figured it out. Tried everything I could imagine ..

**Comment by daytonturner** [comment]
OK - I sorted this out. I had two issues.

1. My plex was not running with proper access to hardware transcoding
2. When i was testing, the stalling I was seeing was coming from transcode happening a couple seconds into the stream when Plex decided to change from Direct Stream to SD (I was playing in the web client), and the transcode was being done in software.  Properly fixing the hardware transcoding issue sped it up by about half, but was still present.  Changing to using a client that would just do direct stream instead of attempting to transcode (Apple TV) made it completely smooth, completely instant.

**Comment by diblaze** [comment]
@daytonturner lovely to hear that you fixed it!
My Plex has correct permissions to HW and I am already playing it in a ATV4K. Sadly still occurring...

**Comment by daytonturner** [comment]
It really helped me to use the plex dashboard while playing the stream - and watching both `top` and `nvidia-smi encodersessions` while watching the stream (and when it stuttered) to identify where the problem was coming from.

My only other piece of advice would be to be absolutely certain that you're able to sustain the bitrate it's trying to push.  

The plex dashboard should tell you if its stalling due to buffering.
